src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -53,8 +53,8 @@ class Table:  # pylint: disable=too-many-public-methods
     catalog: str
     database: str
     name: str
-    object_type: str
-    table_format: str
+    object_type: str | None = None
```

Comment:
Note change on this data type to support `Rule.as_uc_table`

Reply:
this will cause table migration issues. can we get without this change?